styles.css

```diff
@@ -0,0 +1,143 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+.header {
+  background-color: #fff;
+  height: 5rem;
+  width: 100%;
+  position: sticky;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 999;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 10%;
+}
+
+.nav-links ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 50%;
+  list-style-type: none;
+  gap: 2.5rem;
+}
+
+.nav-links a {
+  text-decoration: none;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.logo {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 0.7rem;
+  line-height: 1.25rem;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.ham-menu {
+  display: none;
+}
+
+.main {
+  background-image: url("assets/bg.png");
```

Comment:
- [ ] please, kindly use the correct background image for this section so as to align with the original design.
- [ ] kindly make the gap between the header and the beginning of the headline section smaller. and instead add a little padding under the social icons like in the original design.
-

your design             |  figma
:-------------------------:|:-------------------------:
![Screenshot (898)](https://user-images.githubusercontent.com/65298048/227607352-39847e6b-1abc-40ab-a17b-7db41983e88a.png)  |  ![Screenshot (899)](https://user-images.githubusercontent.com/65298048/227607221-80b0f4d7-7720-406d-a353-183308ad124f.png)